src/clients/firewall_auto.rb

```diff
@@ -87,9 +87,7 @@ def main
         SuSEFirewall.SetStartService(SuSEFirewall.GetEnableService)
       # Import configuration
       elsif @func == "Import"
-        @ret = SuSEFirewall.Import(
-          Convert.convert(@param, :from => "map", :to => "map <string, any>")
-        )
+        @ret = SuSEFirewall.read_and_import(@param)
```

Comment:
We did it in that way to avoid changing the behavior of SuSEFirewall.Import which is public. I fear that someone could use it.